### PR TITLE
Add sklearn tests to test workflow

### PR DESCRIPTION
**Pull request by NathanielRN**
# Description

Adds `sklearn` instrumentation package to the `tox.ini` file so that it can be picked up by the Github actions test workflow

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] Followed the style guidelines of this project
~- [ ] Changelogs have been updated~
~- [ ] Unit tests have been added~
~- [ ] Documentation has been updated~

### CODE REVIEW

**Review by lzchen — APPROVED** [review]

**Review by codeboten — CHANGES_REQUESTED** [review]
Based on the errors in the pypy tests, i'm guessing sklearn isn't supported for pypy? Maybe @crflynn can confirm here.

File: tox.ini [hunk 1]
```diff
@@ -121,6 +121,10 @@ envlist =
     py3{5,6,7,8}-test-instrumentation-celery
     pypy3-test-instrumentation-celery
 
+    ; opentelemetry-instrumentation-sklearn
+    py3{5,6,7,8}-test-instrumentation-sklearn
+    pypy3-test-instrumentation-sklearn
```
Comment (crflynn) [inline tox.ini:126]: sklearn isn't available on pypy so you can remove this

Comment (NathanielRN) [reply tox.ini]: Awesome, thank you! @crflynn

**Review by codeboten — APPROVED** [review]
Thanks for updating!

**Pull request merged**